recipes/cpp-lazy/all/test_package/conanfile.py

```diff
@@ -14,6 +14,8 @@ def layout(self):
 
     def requirements(self):
         self.requires(self.tested_reference_str)
+        # cpp-lazy>=8.0.0 requires fmt library when neither LZ_STANDALONE or LZ_MODULE_EXPORT are not defined
```

Comment:
Looks like so to me too @toge :)

Reply:
@jwillikers  @RubenRBS 
Sorry for the late reply.
If the user has defined LZ_STANDALONE and LZ_MODULE_EXPORT, cpp-lazy does not require the fmtlib.
Once LZ_MODULE_EXPORT is defined, cpp-lazy will require the C++ module.
Since cpp-lazy supports c++11 or later, it is not a portable test.
If we want to keep the portability of cpp-lazy, I think we would require fmtlib only in tests.

Reply:
Hi! Lets bring this PR back to life! 
Which is the minimum supported standard for fmt? I guess if it is greater than C++11 this make sense.